src/tests/Common/dir.common.props

```diff
@@ -1,5 +1,6 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="../../coreclr/Directory.Build.props" />
+  <Import Condition="'$(RuntimeFlavor)' != 'Mono'" Project="../../coreclr/Directory.Build.props" />
+  <Import Condition="'$(RuntimeFlavor)' == 'Mono'" Project="../../mono/Directory.Build.props" />
```

Comment:
I'm surprised that this work as `RuntimeFlavor` shouldn't yet be defined at this point. It gets set when importing the root Directory.Build.props file which imports Subsets.props: https://github.com/dotnet/runtime/blob/2e42b6ea2ac648f02081380bc82ede45160364bd/eng/Subsets.props#L90-L96

Maybe this only works when setting RuntimeFlavor as a global property?

Reply:
If I am not mistaken it comes from the build script via `__RuntimeFlavor` global property: https://github.com/dotnet/runtime/blob/c47a53de9f37b648d26f970cafa54dd24b689ba8/src/tests/build.sh#L387

Reply:
Didn't notice few lines down: https://github.com/dotnet/runtime/pull/111798/files#diff-737ce3ff26a284d7de3cb2d6aad79bac221bc5b3b57a63721e4cbf0e25459a51R16
coreclr is again explicitly set..